xbmc/cores/VideoPlayer/Process/ProcessInfo.h

```diff
@@ -110,4 +114,5 @@ class CProcessInfo
   // player states
   CCriticalSection m_stateSection;
   bool m_stateSeeking;
+  std::atomic_int m_levelVQ;
```

Comment:
What about if RP doesn't use levels and doesn't call SetLevelVQ()? does the GUI know not to call GetLevelVQ()?